lib/companies/schema/company.ex

```diff
@@ -1,27 +1,33 @@
 defmodule Companies.Schema.Company do
   @moduledoc false
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
   alias Companies.Industries
 
-  # @enforce_keys [:industry, :last_changed_on, :location, :name, :website]
+  schema "companies" do
+    field :blog, :string
+    field :description, :string
+    field :github, :string
+    field :industry, :string
+    field :jobs, {:array, :string}
+    field :last_changed_on, :date
+    field :location, :map
```

Comment:
Also possibly a nested struct with it's own validation.

Reply:
Agreed. Location has definitely been a toughy.